examples/aws/tidb-cluster.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v4.0.2
```

Comment:
Suggest adding the following parameters:
configUpdateStrategy: RollingUpdate
nodeSelector for each component

Reply:
nodeSelector is not necessary as we already use `dedicated` taint

Reply:
`configUpdateStrategy: RollingUpdate` is added

Reply:
If not adding `nodeSelector`, we have to add taint to the node group for operator and monitor which we do not do that now and also have to make it clear how to add toleration for operator and monitor.

Reply:
in the old aws terraform scripts, we already use `taints` to reverse node groups for our components (pd, tikv, etc.)

what I did is to remove redundant node selector configuration

other components, tidb-operator are deployed in control node group (in my new docs, I named it `admin`)